### PR TITLE
Adds the ability to set shops to active / disabled

### PR DESCRIPTION
**Pull request by mikemurray**
Fixes items 2 and 3 for https://github.com/reactioncommerce/reaction/issues/2864

This PR adds a simple table to set marketplace shops to `active` / `disabled`

- As a customer
If a shop is disabled, its products are de-listed from the product grid if using the default `Products` publication.

- As an admin
You can see all shops and all products from all shops at the moment.

**This is the start of a full-on marketplace shop management tools.**

### CODE REVIEW

File: imports/plugins/included/marketplace/client/components/marketplaceShops.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  }
+
+  renderShopsTable() {
+    const fields = ["name", "emails", "workflow"];
+
+    const columnMetadata = fields.map((field) => {
+      return {
+        Header: <Components.Translation i18nKey={`marketplaceShops.headers.${field}`} defaultValue={field} />,
+        accessor: field,
+        Cell: (data) => ( // eslint-disable-line
```
Comment (spencern) [inline imports/plugins/included/marketplace/client/components/marketplaceShops.js]: What's disabled here? It's a little more clear if we can use `eslint-disable-line rule-name` so future us/others can understand why we disabled and what for

File: imports/plugins/included/marketplace/server/methods.js [hunk 2]
```diff
@@ -1,23 +1,43 @@
...
+  SHOP_WORKFLOW_STATUS_ACTIVE,
+  SHOP_WORKFLOW_STATUS_DISABLED
+];
+
+export function marketplaceUpdateShopWorkflow(shopId, workflowStatus) {
+  check(shopId, String);
+  check(workflowStatus, String);
+
+  if (shopId === Reaction.getPrimaryShopId()) {
+    throw new Meteor.Error(403, "Cannot change shop status");
```
Comment (spencern) [inline imports/plugins/included/marketplace/server/methods.js]: We're no longer using error codes here, this should be `Meteor.Error("access-denied", "Cannot change shop status");

File: imports/plugins/included/marketplace/client/components/marketplaceShops.js (hunk 1 above)
Comment (spencern) [reply imports/plugins/included/marketplace/client/components/marketplaceShops.js]: I'm actually not seeing any lint errors when removing this, so perhaps not needed?

Comment (mikemurray) [reply imports/plugins/included/marketplace/client/components/marketplaceShops.js]: It's usually a react issue of defining more than one component per file.

**Comment by mikemurray** [comment]
@spencern ninja fixed it before I could 👍

**Pull request merged**